src/gui_common/menus/PlanetSettings.cs

```diff
@@ -106,6 +133,36 @@ public void SetWorldClimateInstability(int index)
         worldClimateInstabilityButton.Selected = index;
     }
 
+    public void SetSulfideLevel(int index)
```

Comment:
I see now why the `(int)` converion is done... These assume the enum value matches the GUI indexes exactly. I would like to see a safer approach here, where these would be given the enum value in and then they internally convert it to indexes. And this should have big warning comments about the indexes needing to always match what is in the tscn file.